buildSrc/src/main/groovy/io.deephaven.java-test-conventions.gradle

```diff
@@ -0,0 +1,88 @@
+plugins {
+    id 'java'
+    id 'io.deephaven.java-jacoco-conventions'
+}
+
+def testJar = project.tasks.register 'testJar', Jar, { Jar jar ->
+    jar.from project.sourceSets.test.output
+    jar.classifier = 'test'
+}
+
+configurations {
+    testOutput.extendsFrom testRuntimeClasspath
+}
+
+artifacts {
+    testOutput testJar
+    archives testJar
+}
+
+project.tasks.withType(Test).all { Test t ->
+    t.with {
+        t.defaultCharacterEncoding = 'UTF-8'
+
+        onlyIf { TestTools.shouldRunTests(project) }
+
+        useJUnit()
+
+        new File("$rootDir/tmp/workspace".toString()).mkdirs()
+        new File("$rootDir/tmp/logs".toString()).mkdirs()
+
+        // pass -PforceTest=true to run test again after they've passed once,
+        // and input source has not changed (i.e. invalidate caching)
+        if (findProperty('forceTest') == 'true') {
+            // not gonna stay uptodate very well like this: :-)
+            inputs.property("forceTest", UUID.randomUUID().toString())
+        }
+
+        enableAssertions = true
+        if (!maxHeapSize) {
+            maxHeapSize = '3g'
+        }
+
+        if (!maxParallelForks) {
+            maxParallelForks = 4
+        }
+
+        if (!forkEvery) {
+            // NB: This comment may be outdated:
```

Comment:
worth removing rather than copy/pasting if we aren't sure it is accurate?

Reply:
I'm running a test to see if this hack is still necessary - https://github.com/devinrsmith/deephaven-core/actions/runs/1849663443 . If these tests fail, or a lot longer than normal to run, I'd say the comment is valid and we need to leave the comment/logic (likely, follow up w/ a ticketed issue).

Reply:
conclusion here? the linked build is stuck on the enterprise plugin